### PR TITLE
project scaffold changes

### PR DESCRIPTION
**Pull request by turb0c0w**
*Issue:*https://github.com/bcgov/entity/issues/

*Description of changes:*
- ui changes
- ui test changes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-6-rbhzjchl.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-6-rbhzjchl.web.app

File: strr-web/nuxt.config.ts [hunk 1]
```diff
@@ -59,7 +64,8 @@ export default defineNuxtConfig({
       payApiURL: `${process.env.VUE_APP_PAY_API_URL || ''}${process.env.VUE_APP_PAY_API_VERSION || ''}`,
       btrApiURL: `${process.env.VUE_APP_BTR_API_URL || ''}${process.env.VUE_APP_BTR_API_VERSION || ''}`,
```
[the remark this answers is not in the document]
Comment (trevoratindustrio) [reply strr-web/nuxt.config.ts:65]: @samIndustrio please include in any update you are in the middle of

File: strr-web/stores/account.ts [hunk 2]
```diff
@@ -69,6 +72,26 @@ export const useBcrosAccount = defineStore('bcros/account', () => {
...
+          statusCode: error?.response?.status || StatusCodes.INTERNAL_SERVER_ERROR,
+          message: error?.response?.data?.message,
+          category: ErrorCategoryE.ACCOUNT_LIST
+        })
+      })
+  }
+
   /** Get the user's account list */
   async function getUserAccounts (keycloakGuid: string) {
     const apiURL = useRuntimeConfig().public.authApiURL
```
[the remark this answers is not in the document]
Comment (trevoratindustrio) [reply strr-web/stores/account.ts:97]: @kris-daxiom the /users/{kcguid_id}/orgs (similar to /settings used earlier), results in a 404.  Perhaps there is a better endpoint for this though to find details of orgs - we are currently hitting the /orgs/{org_id} endpoint to fetch the mailing addresses per Org.  We have discussed to in a future sprint move this handling to the STRR api regardless.

File: strr-web/pages/account-select.vue [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+const query = useRoute().query
+
+let userAccounts = activeUserAccounts
+
+if ('test' in query && query.test === 'true') {
+  const testData: AccountI[] = testAccounts as unknown as AccountI[]
+  userAccounts = testData
+}
+
+const existingAcccountsTitle = `${t('account.existing-account-section.title')} (${userAccounts.length})`
```
[the remark this answers is not in the document]
Comment (trevoratindustrio) [reply strr-web/pages/account-select.vue:39]: @samIndustrio with your ongoing change

File: strr-web/pages/account-select.vue [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+const alertFlavour: AlertsFlavourE = AlertsFlavourE.INFO
+
+const { activeUserAccounts } = useBcrosAccount()
+
+const query = useRoute().query
+
+let userAccounts = activeUserAccounts
+
+if ('test' in query && query.test === 'true') {
```
[the remark this answers is not in the document]
Comment (trevoratindustrio) [reply strr-web/pages/account-select.vue:34]: @samIndustrio with your ongoing change

File: strr-web/stores/account.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (trevoratindustrio) [reply strr-web/stores/account.ts:97]: Oh perfect @kris-daxiom we will make changes in upcoming work mentioned above to adapt to use this endpoint, thanks

**Pull request merged**